tests/Unit/LoanServiceTest.php

```diff
@@ -287,8 +287,8 @@ public function testServiceCanRepayMultipleScheduledRepayments()
         $this->assertDatabaseHas('scheduled_repayments', [
             'id' => $scheduledRepaymentTwo->id,
             'loan_id' => $loan->id,
-            'amount' => 1667,
-            'outstanding_amount' => 333, // 2000 - 1667
+            'amount' => 1666,
+            'outstanding_amount' => 1332, // 1666 - (2000 - 1666)
```

Comment:
The `outstanding_amount` should be (as per my understanding) equal to the initial amount minus the extra amount that the user had paid. Please let me know if this understanding is wrong and some other logic needed to be implemented here.